src/ray/core_worker/test/util.h

```diff
@@ -0,0 +1,28 @@
+#ifndef RAY_CORE_WORKER_TEST_UTIL_H
+#define RAY_CORE_WORKER_TEST_UTIL_H
+
+#include <string>
+
+namespace ray {
+
+bool WaitForCondition(std::function<bool()> condition, int timeout_ms) {
```

Comment:
1) nit, put this in `src/ray/util/test_util.h`, so it can be used by other tests as well. 
2) document

Reply:
sounds good. thanks